manifests/ca.pp

```diff
@@ -150,7 +150,7 @@
         require => File["${etc_directory}/openvpn/${name}/easy-rsa"],
       }
 
-      if $openvpn::params::link_openssl_cnf == true {
+      if $openvpn::link_openssl_cnf == true {
```

Comment:
nitpick: if you want to clean it up, you could do:
```suggestion
      if $openvpn::link_openssl_cnf {
```

But that is really not required